build_tools/bazel_to_cmake/bazel_to_cmake_targets.py

```diff
@@ -86,7 +86,6 @@
     ],
     "@mlir-hlo//:map_lmhlo_to_scalar_op": [
         "tensorflow::external_mhlo_includes",
-        "LmhloDialect",  # Unfortunate.
```

Comment:
Are you sure isn't needed? It looks like this pass really does depend on lmhlo. I'd guess that if this was working without it's just getting picked up by an indirect dependency. Looks like we're dropping the only use of this dependency, so maybe let's just drop this dep from the conversions entirely instead of trying to guess at what exactly it maps to

Reply:
We do not use Lmhlo directly from what I can tell, and would think that if some other targets needs it it should link with it on its own.

Reply:
I agree that if something else needs it, it should link it. I'm saying the opposite: that this target itself definitely uses lmhlo and that if removing this dep doesn't break anything it's only because whatever we're building that links this also links lmhlo some other way. Anyway, I think the right thing to do is to delete this entry entirely, since this target is now unused and should never be used (lmhlo).

Reply:
Ah, I see what you mean now. Thanks for clarifying!

Reply:
I removed it now